ui/src/translations/en.json

```diff
@@ -881,6 +881,12 @@
     "curl": {
       "command": "cURL command",
       "note": "Note that for SECRET and FILE input type, the command must be accommodate to match the real values."
+    },
+    "logs_view": {
+      "raw": "Raw view",
+      "raw_details": "Show full logs in datetime order, but with less features",
+      "compact": "Compact view",
+      "compact_details": "Show compact logs group by Task, but may display less logs"
```

Comment:
```suggestion
      "compact_details": "Show only task logs"
```

Reply:
If you don't explain to people that the "compact" way has less logs, then they will think there is a bug when switching from both

Reply:
what if you point to that in the initial DEBUG log when starting the WaitFor task? sth like "The default log view will only show logs for the last iteration. To see logs for all loop iterations, switch to the temporal view."